swagger2.0/oic.r.cloudopenapi.swagger.json

```diff
@@ -722,49 +722,50 @@
       }
     },
     "DeviceWithLinks": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/Device"
+      "type": "object",
+      "properties": {
+        "device": {
+          "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.d.swagger.json#/definitions/Device"
         },
-        {
+        "devicestatus": {
           "type": "object",
           "properties": {
-            "links": {
-              "type": "array",
-              "items": {
-                "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.res.swagger.json#/definitions/oic.oic-link"
-              }
+            "s": {
+              "$ref": "#/definitions/DeviceStatus"
             }
+          }
+        },
+        "links": {
+          "type": "array",
+          "items": {
+            "$ref": "https://raw.githubusercontent.com/ondrejtomcik/coreocf/Bugzilla-2709/swagger2.0/oic.wk.res.swagger.json#/definitions/oic.oic-link"
+          }
+        }
+      },
+      "example": {
+        "device": {
+          "dmn": "Open Connectivity Foundation",
+          "n": "Food safety sensor",
+          "di": "53080a4f-5e3e-4291-802f-3436238232d2"
+        },
+        "devicestatus": {
+          "s": "online"
+        },
+        "links": [
+          {
+            "href": "/53080a4f-5e3e-4291-802f-3436238232d2/humidity",
+            "rt": [
+              "oic.r.humidity"
+            ]
           },
-          "required": [
-            "rt",
-            "href"
-          ],
-          "example": {
+          {
+            "href": "/53080a4f-5e3e-4291-802f-3436238232d2/temperature",
```

Comment:
As discussed, remove the uuid.

Reply:
Why in this case?  These are the links from the RD, not from the end Device.

Reply:
We have to then discuss bug #2956 again with Dongjoo. It was raised because he wanted to have one type of resource URI. As the batch response from the device is as is - without the deviceId I assume he want to have it also here in the links property without device ids. What make sense.

On the last meeting we discussed, that the device id is there to prevent potential ambiguity if different instances of Devices from the same vendor publish their resources.
But in case of the Open API, we are on the path `http://{domain}/api/v1/devices/{deviceId}/` + `{linkHref}`. The Open API requires that the device specific resource is accessed through the `/{deviceId}/{linkHref}` path. Therefore I don't see necessary a reason, why link should contain the device id in case of the Open API, as the Open API by design distinguish between devices.